src/containers/BooksForm.js

```diff
@@ -1,19 +1,52 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../actions/index';
 
 function BooksForm() {
```

Comment:
### [Optional]
- It would be better if we use ES6 syntax instead of the function keyword. We are using `const` which is ES6 so it would be better if we follow with the consistency